datastore/providers/milvus_datastore.py

```diff
@@ -91,133 +94,173 @@ class Required:
     ),
 ]
 
+# V2 schema, remomve the "pk" field
+SCHEMA_V2 = SCHEMA_V1[1:]
+SCHEMA_V2[4][1].is_primary = True
+
 
 class MilvusDataStore(DataStore):
     def __init__(
         self,
         create_new: Optional[bool] = False,
-        index_params: Optional[dict] = None,
-        search_params: Optional[dict] = None,
+        consistency_level: str = "Bounded",
```

Comment:
consistency_level is missing from the ZillizDatastore init, causing errors in creation.

Reply:
I have added a line "self._consistency_level = "Bounded"" in the ZillizDatastore.__init__